helm-charts/Makefile

```diff
@@ -0,0 +1,52 @@
+PROJECT_NAME=helm-charts
+
+RELEASE_VERSION ?= latest
+CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ../release.version | sed 's/\([0-9.]*\).*/\1/')
+CHART_NAME=strimzi-kafka-operator
+CHART_PATH=./$(CHART_NAME)/
+CHART_RENDERED_TEMPLATES_TMP=../target/charts
+CHART_RENDERED_TEMPLATES_EXAMPLES=../examples/install/cluster-operator/
+
+helm_clean:
+	rm -rf $(CHART_RENDERED_TEMPLATES_TMP)
+	rm -rf strimzi-$(RELEASE_VERSION)/charts/
+	#rm -rf $(CHART_RENDERED_TEMPLATES_EXAMPLES)
+	rm -f $(CHART_NAME)-*.tgz
+
+helm_lint:
+	# Linting Helm Chart
+	helm lint --debug $(CHART_PATH)
+
+helm_template:
+	# Generating rendered template files to: $(CHART_RENDERED_TEMPLATES_TMP)
+	mkdir -p $(CHART_RENDERED_TEMPLATES_TMP)
+	helm template --output-dir $(CHART_RENDERED_TEMPLATES_TMP) --set imageTagOverride=$(RELEASE_VERSION) $(CHART_PATH)
+
+helm_examples: helm_template
```

Comment:
This looks like it generates even the CRD definitions fro the Helm Chart. Should we then disable their generating in the `api` Maven?

Reply:
The generation in `api/pom.xml` generates to `helm-charts` and this `helm_template` then renders the charts into `examples`

Reply:
It IMHO generates both: https://github.com/strimzi/strimzi-kafka-operator/blob/4e33a84c7da846de9e4da4af5303c526c59e2244/api/pom.xml#L112